core/templates/pages/topic-editor-page/modal-templates/create-new-subtopic-modal.component.spec.ts

```diff
@@ -117,9 +117,9 @@ describe('create new subtopic modal', function() {
     ngbActiveModal = TestBed.inject(NgbActiveModal);
     TestBed.inject(WindowRef);
     topicUpdateService = TestBed.inject(TopicUpdateService);
-    topicEditorStateService =
+    topicEditorStateService = (
```

Comment:
Is the `as unknown` on line below still needed?

Reply:
Removed, use `useValue`.